packages/edge-gateway/wrangler.toml

```
@@ -105,11 +103,17 @@ name = "PUBLIC_RACE_STATUS_CODE"
 # Test!
 [env.test]
 workers_dev = true
-kv_namespaces = [{ binding = "DENYLIST" }]
+
+[[env.production.services]]
+binding = "CID_VERIFIER"
+type = "service"
+service = "cid-verifier-api"
```

```suggestion
service = "dotstorage-cid-verifier-production"
```

and staging on the other one